cmd/invoke.go

```diff
@@ -79,11 +82,22 @@ dapr invoke --unix-domain-socket --app-id target --method sample --verb GET
 			}
 		}
 
-		response, err := client.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb, invokeSocket)
-		if err != nil {
-			err = fmt.Errorf("error invoking app %s: %w", invokeAppID, err)
-			print.FailureStatusEvent(os.Stderr, err.Error())
-			os.Exit(1)
+		var response string
+		if kubernetesMode {
```

Comment:
Given that both kubernetes mode and standalone mode have same error handling, can we follow DRY and do this instead?
```go
if kubernetesMode {
    response, err = kubernetes.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb)
} else {
    client := standalone.NewClient()
    response, err = client.Invoke(invokeAppID, invokeAppMethod, bytePayload, invokeVerb, invokeSocket)
}

// now handle error and response
if err != nil {
    // handle error
}
```

Reply:
Thank you for review. I fixed it.